src/Microsoft.Extensions.CodeGeneration.Utils/CommonUtilities.cs

```diff
@@ -22,7 +22,7 @@ public static class CommonUtilities
             {
                 using (var pdb = new MemoryStream())
                 {
-                    if (PlatformHelper.IsMono)
+                    if (PlatformHelper.IsMono || !PlatformHelper.IsWindows)
```

Comment:
We probably don't care about the pdbs for scaffolded items in general. I think it should be ok to remove the else entirely.